automatic/qbittorrent/tools/chocolateyinstall.ps1

```diff
@@ -1,5 +1,10 @@
 ﻿$ErrorActionPreference = 'Stop'
 
+$is32bit = Get-OSArchitectureWidth
+if ($is32bit -eq '32') {
+  Write-Host "WARNING! qBittorrent is no longer available in 32-bit after version 4.4.5, pin the package version to 4.4.5 with command ``choco pin add --name=`"'qbittorrent'`" --version=`"'4.4.5'`"``"
+}
+
```

Comment:
Right now, this won't halt the package installation on an 32 bit environment - and, actually, as :14 still exists (and won't be updated), this could lead to confusion if the file is not added by `update.ps1`.

We should change this to be a throw, and entirely remove the remnants of the x86 installer (the `file` in the `$packageArgs` block).

As a sidenote, as well, `Get-OSArchitectureWidth` has a parameter for comparisons, which returns a bool.

```suggestion
if (Get-OSArchitectureWidth -Compare 32) {
  throw "qBittorrent is no longer available in 32-bit after version 4.4.5, pin the package version to 4.4.5 with command ``choco pin add --name=`"'qbittorrent'`" --version=`"'4.4.5'`"``"
}

```

Reply:
Apologies for missing this @gobalius @JPRuskin